### PR TITLE
fix: title js css

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: static/app/views/performance/browser/resources/index.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ function ResourcesLandingPage() {
         <StyledTabs>
           <TabList hideBorder>
             <TabList.Item key="resource.css/script">
-              {t('Javascript/Stylesheets')}
+              {t('JavaScript/Style sheets')}
```
Comment (cleptric) [inline static/app/views/performance/browser/resources/index.tsx:88]:
```suggestion
              {t('JavaScript/Style Sheets')}
```

Title case ftw

**Review by cleptric — APPROVED** [review]

**Review by DominikB2014 — APPROVED** [review]
Thanks for addressing that!

**Comment by DominikB2014** [comment]
@bruno-garcia fyi, I think we might try JS/CSS in the future.

File: static/app/views/performance/browser/resources/index.tsx [hunk 2]
```diff
@@ -85,7 +85,7 @@ function ResourcesLandingPage() {
         <StyledTabs>
           <TabList hideBorder>
             <TabList.Item key="resource.css/script">
-              {t('Javascript/Stylesheets')}
+              {t('JavaScript/Style Sheets')}
```
Comment (bruno-garcia) [inline static/app/views/performance/browser/resources/index.tsx:88]:
```suggestion
              {t('JS/CSS')}
```

**Comment by bruno-garcia** [comment]
> @bruno-garcia fyi, I think we might try JS/CSS in the future.

done

**Pull request merged**